src/pybind/mgr/nfs/export.py

```diff
@@ -185,7 +190,7 @@ def nfs_rados_configs(rados: 'Rados', nfs_pool: str = POOL_NAME) -> Set[str]:
     return ns
 
 
-class ExportMgr:
+class ExportMgr(CephfsClient):
```

Comment:
You'd need to call the base class ctor:

```
super(ExportMgr, self).__init__(mgr)
```

or if to do it without subclass
```
self.fs_client =  CephfsClient(mgr)
```

then
```
with open_filesystem(self.fs_client, fs_name) as ...
```

both as ok.

Reply:
I'll add `self.fs_client =  CephfsClient(mgr)` to `__init__` of `ExportMgr`

Reply:
@vshankar make check still fails with the same failure:

```
Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 400, in create_export
    return self.create_cephfs_export(**kwargs)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/nfs/export.py", line 643, in create_cephfs_export
    with open_filesystem(self, fs_name) as fs_handle:
  File "/usr/lib/python3.10/contextlib.py", line 135, in __enter__
    return next(self.gen)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/mgr_util.py", line 351, in open_filesystem
    if fsc.is_stopping():
AttributeError: 'ExportMgr' object has no attribute 'is_stopping'
```

if you notice it says `with open_filesystem(self, fs_name) as fs_handle:` while that line is no more, its now `with open_filesystem(self.fs_client, fs_name) as fs_handle:`, seems like an issue with `make check`?

Reply:
really? can you trigger a fresh make check run and see if it reports the same error again. odd.

Reply:
yeah, i rebased the PR, let's see how it goes

Reply:
created instance in `__init__` (`self.fs_client =  CephfsClient(mgr)`) and its working